grpc/src/compiler/cpp_generator.h

```diff
@@ -38,10 +38,6 @@
 // can be used to generate code for other serialization systems, such as
 // FlatBuffers.
 
-#include <memory>
```

Comment:
We copy this file from the grpc project, so not sure how useful changes here are.

Reply:
This directive is not needed since the comit 2e7806ed by Harsh Vardhan at 08/19/2016.
This commit was removed the dependency on `std::unique_ptr<>`.